projects/packages/connection-ui/src/class-admin.php

```diff
@@ -16,8 +16,11 @@ class Admin {
 	 * Construction.
 	 */
 	public function __construct() {
-		add_action( 'admin_menu', array( $this, 'register_submenu_page' ), 1000 );
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+		if ( ! did_action( 'jetpack_on_connection_ui_init' ) ) {
+			add_action( 'admin_menu', array( $this, 'register_submenu_page' ), 1000 );
+			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+			do_action( 'jetpack_on_connection_ui_init' );
```

Comment:
Do you think you could add a docblock here for this new hook? The docblocks make it possible to extract data into a codex, and may be useful for our future selves needing to use that hook. :)

Reply:
Ok, that's why we don't use auto-merge, got it 🙂

Reply:
That's a great idea, let's do that over in https://github.com/Automattic/jetpack/pull/19846 where I had a question about the `@since` bit.